dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +689,299 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.Must(uuid.NewV4()).String()[:8]
+	datastoreNamespace := "golang-samples" + u
+	datastoreKind := "task"
+
+	if err := inspectDataStoreSendToScc(&buf, tc.ProjectID, datastoreNamespace, datastoreKind); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Job created successfully:"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+func TestInspectDataToHybridJobTrigger(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	trigger, err := createJobTriggerForInspectDataToHybridJobTrigger(t, tc.ProjectID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fmt.Print("Name:" + trigger)
+	if err := inspectDataToHybridJobTrigger(&buf, tc.ProjectID, "My email is test@example.org and my name is Gary.", trigger); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "successfully inspected data using hybrid job trigger"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectDataToHybridJobTrigger got %q, want %q", got, want)
+	}
+	if want := "Findings"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectDataToHybridJobTrigger got %q, want %q", got, want)
+	}
+	if want := "Job State: ACTIVE"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectDataToHybridJobTrigger got %q, want %q", got, want)
+	}
+	if want := "EMAIL_ADDRESS"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectDataToHybridJobTrigger got %q, want %q", got, want)
+	}
+	if want := "PERSON_NAME"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectDataToHybridJobTrigger got %q, want %q", got, want)
+	}
+
+	deleteActiveJob(t, tc.ProjectID)
+}
+
+func deleteActiveJob(t *testing.T, project string) error {
+	t.Helper()
+	ctx := context.Background()
+	client, err := dlp.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+	// jobTrigger := fmt.Sprint("projects/", project, "/locations/global/jobTriggers/", jobTriggerId)
+	req := &dlppb.ListDlpJobsRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/global", project),
+		Filter: "state=ACTIVE",
```

Comment:
Will this end up deleting all active jobs? What if there are other CI pipeines running in parallel?

Reply:
Updated the filter with the trigger name, so that it doesn't cause other jobs.